.github/workflows/snyk.yml

```diff
@@ -0,0 +1,22 @@
+# ref: https://github.com/snyk/actions/tree/master/gradle-jdk11
+name: snyk for gradle
+on:
+  push:
+  release:
+    types: 
+      - created
+  pull_request_target:
+jobs:
+  security:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Run Snyk to check for vulnerabilities
+      uses: snyk/actions/gradle-jdk11@master
```

Comment:
Can we use a specific version instead of master - `snyk/actions/gradle-jdk11@master`?

Reply:
yup. latest it 0.3.0 will update and test